### ISSUE TITLE
Questers spew out errors

### ISSUE THREAD

**Issue by defaultsucks** [CitizensDev/Citizens#82]
I've recently requested an example quests.yml file and I was linked to this: http://dev.bukkit.org/paste/3884/
I've pasted it into my quests.yml, reloaded citizens. Everything loaded just fine. When I assign any quest to a quester and left/right click on them, the console spews out a bunch of errors like this:

```
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
at net.citizensnpcs.questers.Quester.updateDescription(Quester.java:171)
at net.citizensnpcs.questers.Quester.cycle(Quester.java:163)
```

Nothing else happens. The quest information isn't displayed, it just shows these errors in the console. I've tried messing with different things, making it so after the commented out lines at the top there's only one quest called example, removing single quotes in all the text except the tags before the : .It still spews out the same errors. This is on citizens 1.1 dev build #119

**Comment by havejack** [comment]
Questers are not done atm so expect bugs

**Comment by defaultsucks** [comment]
I know, it's just that since the example file is out and people say quests do function at least to a minimum degree I thought I'd still report this issue to help as much as I can.

**Comment by defaultsucks** [comment]
This is fixed in the dev build #120
